relay-server/src/utils/metrics_rate_limits.rs

```diff
@@ -106,6 +106,21 @@ pub struct TransactionCount {
     pub has_profile: bool,
 }
 
+pub fn get_transaction_and_profile_count(
```

Comment:
This function already kind-of exists in the processors `create_metrics_item` and uses `SourceQuantities` to count, we can re-use this instead.

Reply:
alright! will check that. 

unrelated to this pr, but i guess the same logic applies to MetricsLimiter::create?